canvas/biplot.py

```diff
@@ -0,0 +1,167 @@
+from __future__ import division
+import matplotlib
+import matplotlib.pyplot as plt
+import matplotlib.patches as mpatches
+import numpy as np
+import pandas as pd
+from skbio.stats.composition import clr, closure, centralize,
+from skbio.stats.composition import multiplicative_replacement, perturb_inv
+from biom import load_table
+from functools import partial
+import math
+
+
+def make_biplot(sample_df,
+                feature_df,
+                sample_df_metadata=None,
+                feature_df_metadata=None,
+                sample_color_category=None,
+                feature_color_category=None,
+                sample_color_dict=None,
+                feature_color_dict=None,
+                eigenvalue=None):
+
```

Comment:
Can we add some numpy-docs here?  Here is a link to the [numpy doc-strings](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt)

 I think if you copy the following right here, that should be good enough.

```python
""" Creates a biplot.

Creates a 2D biplot where the points correspond to samples and the
vectors correspond to features.  The distance between points often 
correspond to sample dissimilarity and the angle between vectors 
is often a proxy for correlations between vectors. 

Parameters
----------
samples : pd.DataFrame
    A table of samples by principal coordinates.
features: pd.DataFrame
   A table of features by principal coordinates.
sample_metadata: pd.DataFrame
   An optional table for specifying metadata for samples
   to use in plotting the sample points.
feature_metadata: pd.DataFrame
   An optional table for specifying metadata for features
   to use in plotting the feature vectors.
sample_color_category: str
  Name of variable in `sample_metadata`.
sample_color_category: str
  Name of variable in `sample_metadata`.
feature_color_category: str
  Name of variable in `feature_metadata`.
sample_color_dict: dict
  Dictionary of colors corresponding to different sample
  categories in `sample_metadata`.
feature_color_dict: dict
  Dictionary of colors corresponding to different feature
  categories in `feature_metadata`.
eigenvalues: pd.Series
  A vector of eigenvalues along the principal coordinates.

Returns
-------
matplotlib.figure.Figure
    Container for biplot figure
iterable of matplotlib.axes._subplots.AxesSubplot
   Two matplotlib axes objects.  The first axes object contains
   plotting information about the samples.  The second axes 
   object contains plotting information about the features.
"""
```